2021_JuMP/slides/marek.tex

```diff
@@ -0,0 +1,201 @@
+\begin{frame}{Group Theory to the rescue!}
+  \begin{block}{Definition}
+    A \textbf{group} $(G, *)$ is a set closed under the multiplication $*$, together with the existence of unique inverse elements.
+  \end{block}
+    {\small Example: The set of all bijections of the set $\{1,\ldots n\}$ with composition is a group (known as the full symmetric group on $n$ letters $S_n$).}
+    
+    $S_2 = \{(1)(2), (1,2)\}$ \textbf{acts} on polynomial ring $\mathbb{R}[x]$ by
```

Comment:
You might want to introduce the notation `(1,2)`, `(1)(2)`.

Reply:
yeah, I was mostly thinking about explaining this in the talk; it doesn't have to be on the slide, does it?

Reply:
No, if you say it it's good